pkg/kine/server/create.go

```diff
@@ -28,13 +36,18 @@ func (l *LimitedServer) create(ctx context.Context, put *etcdserverpb.PutRequest
 		return nil, unsupported("prevKv")
 	}
 
+	createCnt.Add(ctx, 1)
+
 	rev, err := l.backend.Create(ctx, string(put.Key), put.Value, put.Lease)
+	span.SetAttributes(attribute.Int64("revision", rev))
 	if err == ErrKeyExists {
+		span.RecordError(err)
```

Comment:
This shouldn't be recorded as it is not a misbehavior.

Reply:
This part is removed.